<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitProgramList">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>310</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="vboxProgramList">
        <item>
         <widget class="QTableView" name="tblPrograms">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hboxProgramBtns">
          <item>
           <widget class="QPushButton" name="btnProgramAdd">
            <property name="toolTip">
             <string>Create a new program</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="list-add"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnProgramDel">
            <property name="toolTip">
             <string>Remove selected program</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="list-remove"/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitProgramPanes">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitSplitSelect">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_1">
         <property name="maximumSize">
          <size>
           <width>310</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="vboxLayerList">
          <item>
           <widget class="QTableView" name="tblLayers">
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hboxLayerBtns">
            <item>
             <widget class="QPushButton" name="btnLayerAdd">
              <property name="toolTip">
               <string>Add a new layer</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-add"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnLayerDel">
              <property name="toolTip">
               <string>Remove selected layer</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-remove"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnLayerEdit">
              <property name="toolTip">
               <string>Edit layer</string>
              </property>
              <property name="text">
               <string>Edit</string>
              </property>
              <property name="icon">
               <iconset theme="entry-edit"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <layout class="QVBoxLayout" name="vboxSplitList">
          <item>
           <widget class="QTableView" name="tblSplits">
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hboxSplitBtns">
            <item>
             <widget class="QPushButton" name="btnSplitAdd">
              <property name="toolTip">
               <string>Create a new split</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-add"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSplitDel">
              <property name="toolTip">
               <string>Remove selected split</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-remove"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnSplitPlay">
              <property name="toolTip">
               <string>Play split audio</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="media-playback-start"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnToneImport">
              <property name="toolTip">
               <string>Change split audio</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="document-open"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnToneExport">
              <property name="toolTip">
               <string>Save split audio</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="document-save-as"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSplitEdit">
              <property name="toolTip">
               <string>Edit split</string>
              </property>
              <property name="text">
               <string>Edit</string>
              </property>
              <property name="icon">
               <iconset theme="entry-edit"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="PianoKeyboardWidget" name="keyMap" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExportAs">
     <property name="title">
      <string>E&amp;xport As...</string>
     </property>
     <addaction name="actionSoundFont2"/>
    </widget>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuExportAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
    <addaction name="actionVelocities"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionGuessToneSize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut"/>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSoundFont2">
   <property name="text">
    <string>&amp;SoundFont 2</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-undo"/>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-redo"/>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="window-close"/>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset theme="document-properties"/>
   </property>
   <property name="text">
    <string>Bank &amp;Properties</string>
   </property>
  </action>
  <action name="actionVelocities">
   <property name="text">
    <string>&amp;Velocities</string>
   </property>
  </action>
  <action name="actionGuessToneSize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Guess full tone size</string>
   </property>
   <property name="toolTip">
    <string>Disable if you have issues with noise or clicking at the end of tones.
Requires reloading the file to take effect.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PianoKeyboardWidget</class>
   <extends>QWidget</extends>
   <header location="global">guicommon/PianoKeyboardWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
